### DISCUSSION TITLE
Adaptereinstellungen Authentifizierungscode Seriennummer

### DISCUSSION THREAD

**Discussion by Lightyear5595** [DEV2DEV-DE/ioBroker.boschindego#15] [General]
Das Bild in der Beschreibung ist wirklich schön, aber WO finde ich diese Information?

**Comment by mcm1957** [comment]
Dieses Repositoriy ist NICHT das Geimatverzeichnis des Adapter ioBroker.bischindego der im ioBroker Repository gelistet ist.
Das Repository des des Adapter ioBroker.bischindego befindet sich hier: 

https://github.com/iobroker-community-adapters/ioBroker.boschindego
